hive_agent/store/store.py

```diff
@@ -1,15 +1,21 @@
 import logging
 import sys
-from datetime import datetime
+import os
```

Comment:
the tests are failing because an import was removed: `from datetime import datetime`